packages/common/src/TransactionUtils.js

```diff
@@ -39,7 +39,10 @@ const runTransaction = async ({ web3, transaction, transactionConfig, availableA
   if (await accountHasPendingTransactions(web3, transactionConfig.from))
     transactionConfig.nonce = await getPendingTransactionCount(web3, transactionConfig.from);
   // Else, there is no pending transaction and we use the current account transaction count as the nonce.
-  else transactionConfig.nonce = await web3.eth.getTransactionCount(transactionConfig.from);
+  // this method does not play niceley in tests. leave the nounce null to auto fill.
+  else if (argv.network != "test") transactionConfig.nonce = await web3.eth.getTransactionCount(transactionConfig.from);
+
+  console.log("args", argv);
```

Comment:
not sure if this is intended

Reply:
was not. thanks for catching. have removed